alembic-autogenerate.py

```diff
@@ -2,26 +2,26 @@
 
 #
 # How to use this script:
-# 1. Load the LSST environment and setup sdm_schemas and felis.
-#        source loadLSST.bash
-#        setup felis
-#        setup -r /path/to/sdm_schemas
+# 1. Install Felis and sdm_schemas, set environment variables:
+#        pip install lsst-felis testing.postgresql alembic sqlalchemy pyyaml
```

Comment:
This seems to be missing a few dependencies. What I found worked from scratch was:

`pip install lsst-felis testing.postgresql alembic sqlalchemy pyyaml black psycopg2-binary`